src/repository/woo-api/models/products.type.ts

```diff
@@ -33,16 +33,39 @@ export const ProductsSchema = Type.Array(ProductSchema);
 
 export type ProductsFromWooType = Static<typeof ProductsSchema>;
 
+// User receive this product type with id
 export type ProductType = {
-  id: number;
+    id: number;
+    name: string;
+    sku: string;
+    price: string;
+    stock_quantity: number | null;
+    images: {
+      id: number;
+      src: string;
+    }[];
+}
+
+export type ProductsType = ProductType[];
+
+// When user create new product there is no id yet
+export type NewProductType = {
```

Comment:
```suggestion
export type NewProductType = Omit<ProductType, "id">
```

Reply:
Keep this to maybe extend response from Woo later